core/lib/auth/auth/routes.py

```diff
@@ -73,8 +75,8 @@ def create_token(
         raise ValueError(f"Valid account types are 'user' and 'customer', not '{account_type}'")
     if account_type == "user":
         # make sure a user is not given admin privileges
-        if "users:admin" in scope:
-            raise ValueError("User tokens cannot have scope 'users:admin'")
+        if "customer" in scope:
```

Comment:
The scope is a list, not a string, so you'll probably want to check each item in the list